<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>594</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Appointment Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListView" name="appointmentLv"/>
      </item>
      <item>
       <widget class="QListWidget" name="customersServedLw"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="serveBtn">
        <property name="text">
         <string>Serve Customer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="customerNameLb">
        <property name="text">
         <string>Customer Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="customerNameTb">
        <property name="toolTip">
         <string>Enter Customers Name</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="reasonForVistLb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="appointmentLb">
        <property name="text">
         <string>Reason For Visit:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="resonForVistCb">
        <property name="toolTip">
         <string>Enter Reason for visit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="priorityLb">
        <property name="text">
         <string>Priority:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="prioritySb">
        <property name="toolTip">
         <string>Enter Customers Priority</string>
        </property>
        <property name="toolTipDuration">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="submitBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
